controller/api/api.go

```diff
@@ -612,10 +612,19 @@ func (a *Api) scaleContainer(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err := a.manager.ScaleContainer(containerId, numInstances); err != nil {
+	result := a.manager.ScaleContainer(containerId, numInstances)
+	json, err := json.Marshal(result)
+	if err != nil {
```

Comment:
@ehazlett I've added error handling here if we encounter a problem marshalling the result, so we return a 500 error and the message from the error.  Is this okay? :)